apis/atrium-api-cc-en_UK/src/test/kotlin/ch/tutteli/atrium/api/cc/en_UK/CharSequenceAssertionsSpec.kt

```diff
@@ -1,14 +1,15 @@
 package ch.tutteli.atrium.api.cc.en_UK
 
-import ch.tutteli.atrium.verbs.internal.AssertionVerbFactory
 import ch.tutteli.atrium.creating.Assert
+import ch.tutteli.atrium.verbs.internal.AssertionVerbFactory
 
 object CharSequenceAssertionsSpec : ch.tutteli.atrium.spec.integration.CharSequenceAssertionsSpec(
     AssertionVerbFactory,
     Assert<CharSequence>::containsDefaultTranslationOf.name to Assert<CharSequence>::containsDefaultTranslationOf,
     Assert<CharSequence>::containsNotDefaultTranslationOf.name to Assert<CharSequence>::containsNotDefaultTranslationOf,
     Assert<CharSequence>::isEmpty.name to Assert<CharSequence>::isEmpty,
     Assert<CharSequence>::isNotEmpty.name to Assert<CharSequence>::isNotEmpty,
+    Assert<CharSequence>::isNotBlank.name to Assert<CharSequence>::isNotBlank,
```

Comment:
replace by:
```
"${Assert<CharSequence>::isNotBlank.name} from en_GB, not implemented in en_UK" to Assert<CharSequence>::isNotBlank,
```

Reply:
Nah.. will not work, en_GB is not available in en_UK spec. So turn `object CharSequenceAssertionsSpec` into `class CharSequenceAssertionsSpec` and replace line by:
```
"isNotBlank not implemented in en_UK" to Companion::isNotBlank,
```
And add companion object (code not tested, please adopt as required):
```
    companion object {
        private fun isNotBlank(plant: Assert<CharSequence>) : Assert<CharSequence>
            = plant.addAssertion(AssertImpl.charSequence.isNotBlank(plant))
    }
```